test/registrationTest.js

```diff
@@ -6,39 +6,50 @@ chai.use(require('chai-http'));
 
 const app = require('../index.js'); // Our app
 
-var body = {
-    first:"test",
-    last:"test1",
-    username: "fakeuser",
-    email:"test@fake.email.com",
-    password:"test"
+var registrationBody = {
+    "first":"test",
+    "last":"test",
+    "email":"test@fake.email.com",
+    "password":"test"
 };
 
-describe('Registration endpoint', function () {
-
-    this.timeout(5000); // How long to wait for a response (ms)
-
-    before(function () {
-
-    }
-    );
-
-    after(function () {
+var signinBody = {
+    "username":"test",
+    "password":"password"
+}
 
-    }
-    );
+describe('Home page endpoint', function () {
```

Comment:
This is a test to hit the home page endpoint. Just checks if 200 response is recieved